### PR TITLE
nohdmi image support for imx8mqevk

### PR DESCRIPTION
**Pull request by igoropaniuk**

### CODE REVIEW

File: meta-lmp-bsp/dynamic-layers/freescale-layer/recipes-bsp/imx-mkimage/imx-boot_%.bbappend [hunk 1]
```diff
@@ -31,7 +32,28 @@ do_compile_prepend_mx8m() {
     fi
 }
 
+do_compile_append() {
+    if [ "${IMXBOOT_TARGETS}" = "flash_evk_spl" ]; then
+            cp ${BOOT_STAGING}/flash.bin-nohdmi ${S}/${BOOT_CONFIG_MACHINE}-${IMXBOOT_TARGETS}-nohdmi
+    fi
+}
```
Comment (ricardosalveti) [inline meta-lmp-bsp/dynamic-layers/freescale-layer/recipes-bsp/imx-mkimage/imx-boot_%.bbappend:39]: Do we need this do_compile_append block? The file is already created at ${BOOT_STAGING}/flash.bin-nohdmi, so why not simply installing/deploying from there instead?

Comment (igoropaniuk) [reply meta-lmp-bsp/dynamic-layers/freescale-layer/recipes-bsp/imx-mkimage/imx-boot_%.bbappend:39]: just decided to do the same as it's done in the original recipe in `do_compile()` (rename `flash.bin` -> `${BOOT_CONFIG_MACHINE}-${IMXBOOT_TARGETS` )

Comment (ricardosalveti) [reply meta-lmp-bsp/dynamic-layers/freescale-layer/recipes-bsp/imx-mkimage/imx-boot_%.bbappend:39]: Oh, sure, fine then.

Comment (ricardosalveti) [reply meta-lmp-bsp/dynamic-layers/freescale-layer/recipes-bsp/imx-mkimage/imx-boot_%.bbappend:39]: So, looking at the original do-compile block, we might be missing a similar for here.

IMXBOOT_TARGETS can be a list, so we need a for similar to how it is done in do_compile.

Like:

```
do_compile_append() {
    for target in ${IMXBOOT_TARGETS}; do
        if [ "${target}" = "flash_evk_spl" ]; then
            cp ${BOOT_STAGING}/flash.bin-nohdmi ${S}/${BOOT_CONFIG_MACHINE}-${target}-nohdmi
        fi
    done
}
```

And same for install and deploy.

Comment (igoropaniuk) [reply meta-lmp-bsp/dynamic-layers/freescale-layer/recipes-bsp/imx-mkimage/imx-boot_%.bbappend:39]: fixed

**Comment by igoropaniuk** [comment]
@ricardosalveti all comments addressed

**Review by angolini — APPROVED** [review]

**Review by vanmaegima — APPROVED** [review]

**Review by ricardosalveti — APPROVED** [review]
LGTM

**Review by mike-scott — APPROVED** [review]

**Pull request merged**